src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js

```diff
@@ -215,7 +215,7 @@ function TrackScrubberButton({ player, trackScrubberRef, timeToolRef, isPlaylist
       In playlists, markers are timepoint information representing highlighting annotations, 
       therefore omit reading markers information for track scrubber in playlist contexts. 
     */
-    if (player.markers && player.markers.getMarkers()?.length > 0 && !isPlaylist) {
+    if (player.markers && typeof player.makrers !== 'function' && player.markers.getMarkers()?.length > 0 && !isPlaylist) {
```

Comment:
Good catch!

For avalon manifests, we use player markers to highlight the structure ranges. So, I think we need to refactor this code to filter only the structure range markers, now that we have the transcript search hits using markers as well.
I tested the following code with a manifest from avalon, which seemed to work;
```
if (player.markers && typeof player.markers.getMarkers === 'function' && player.markers.getMarkers()?.length > 0 && !isPlaylist) {
      const track = player.markers.getMarkers().filter(m => m.class == 'ramp--track-marker--fragment')[0];
      if (track?.length > 0 && track[0].key != currentTrack?.key) {
        setCurrentTrack(track[0]);
      }
    }
```